website/docs/usage/args.md

```diff
@@ -36,7 +36,7 @@ git-cliff [FLAGS] [OPTIONS] [--] [RANGE]
     --tag-pattern <PATTERN>      Sets the regex for matching git tags [env: GIT_CLIFF_TAG_PATTERN=]
     --with-commit <MSG>...       Sets custom commit messages to include in the changelog [env: GIT_CLIFF_WITH_COMMIT=]
     --skip-commit <SHA1>...      Sets commits that will be skipped in the changelog [env: GIT_CLIFF_SKIP_COMMIT=]
--p, --prepend <PATH>             Prepends entries to the given changelog file [env: GIT_CLIFF_PREPEND=]
+-p, --prepend <PATH>             Prepends entries to the given changelog file, incompatible with -o [env: GIT_CLIFF_PREPEND=]
```

Comment:
I don't think it is required to mention it here. 

```suggestion
-p, --prepend <PATH>             Prepends entries to the given changelog file [env: GIT_CLIFF_PREPEND=]
```

Reply:
We can probably add a comment to the `prepend` example in `examples.md` instead.

Reply:
@orhun I added a line to the examples and removed it from the website docs :)